exercises/04-integration/README.md

```diff
@@ -105,14 +105,14 @@ public class ShipWithFedexCommandHandler : IHandleMessages<ShipWithFedexCommand>
 
     private XDocument CreateFedexRequest(string name, string street, string city, string postalCode, string country)
     {
-        var shipment =
+        var shipment = new XDocument(
             new XElement("FedExShipment",
                 new XElement("ShipTo",
                     new XElement("Name", name),
                     new XElement("Street", street),
                     new XElement("City", city),
                     new XElement("PostalCode", postalCode),
-                    new XElement("Country", country)));
+                    new XElement("Country", country))));
```

Comment:
It's an improvement, but it's still different to what's in the after solution (see next line). We should probably copypasta this from https://github.com/Particular/Workshop/blob/d6a2ca5bb248908b5775a04e1d34e19ed95e3776/exercises/04-integration/after/Divergent.ITOps/Handlers/ShipWithFedexCommandHandler.cs#L36-L48